.github/workflows/tests.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        php: ['7.4', '8.0', '8.1']
+        php: [8.0]
```

Comment:
Looks like it dropped `8.1`. You should probably re-add it. I'll fix that up for the future.

Reply:
Thanks for this! 🥳